.github/workflows/update-labels.yml

```diff
@@ -0,0 +1,18 @@
+name: Update label when user responds
+permissions:
+  issues: write
+
+on:
+  issue_comment:
+    types: [created]
```

Comment:
Do we need to trigger this every comment? how about just like the `close-unresponded.yml`?

Reply:
Running this action on `issue_comment` event is important because the event details are needed on the conditional statement on line 15 (currently).

```yaml
if: ${{ github.event.comment.user.login == github.event.issue.user.login && contains(github.event.issue.labels.*.name, 'wait for update') && !contains(github.event.issue.labels.*.name, 'user responded') }}
```